tagged-localbuild.xml

```diff
@@ -0,0 +1,825 @@
+  <project name="platform/external/wayland-protocols" path="external/wayland-protocols" revision="ef003a5e13cff44a9b9bc0e081e8851378e5e067" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
+  <project name="platform/external/webp" path="external/webp" revision="5b8ffe46a0fbf17592104c3a7d85a1074b868bcc" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk,qcom_msm8x26"/>
+  <project name="platform/external/webrtc" path="external/webrtc" revision="661e84967bb2600427cb65715caeeba112b94244" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
+  <project name="platform/external/wpa_supplicant_8" path="external/wpa_supplicant_8" revision="bb03019c0bbf6f57b839c4a40a2c6c3b130c7607" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
+  <project name="platform/external/wycheproof" path="external/wycheproof" revision="1b808efc7c903b0eaae0da8f6788f55513e4e339" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
+  <project name="platform/external/xmp_toolkit" path="external/xmp_toolkit" revision="44d7e4d00ef6da63a75bf3a4f9296872cc4d2ac1" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
+  <project name="platform/external/xz-embedded" path="external/xz-embedded" revision="51923a98b0af813e3db6a027d64471ee8710a4f2" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
+  <project name="platform/external/xz-java" path="external/xz-java" revision="bd7892bb8927360cd34805bc332039154a2110e0" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
+  <project name="platform/external/yapf" path="external/yapf" revision="45d923c40605a71867d174abcd04a1db76945651" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="vts,projectarch,pdk"/>
+  <project name="platform/external/zlib" path="external/zlib" revision="1a9b432817192342bbfb5a389058a45eea9f816c" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
+  <project name="platform/external/zopfli" path="external/zopfli" revision="4e42ccafb7c571f502a23177bca2d3cfc209035c" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
+  <project name="platform/external/zxing" path="external/zxing" revision="65193280855e46d11bf5b44f1a7e49ec0c7bd57e" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
+  <project name="platform/frameworks/av" path="frameworks/av" revision="c6f7f783292bb5758be1a5c5323d2e68e1f51069" upstream="refs/tags/android-11.0.0_r48" dest-branch="refs/tags/android-11.0.0_r48" groups="pdk"/>
```

Comment:
This hash doesn't exist upstream, because `repo manifest -ro tagged-localbuild.xml` was launched after applying patches.

https://android.googlesource.com/platform/frameworks/av/+/c6f7f783292bb5758be1a5c5323d2e68e1f51069

Same goes for other repos that have been patched.

Reply:
Fixed those after syncing in a separate tree.